GitUI/CommandsDialogs/WorktreeDialog/FormCreateWorktree.cs

```diff
@@ -124,17 +124,17 @@ private void createWorktreeButton_Click(object sender, EventArgs e)
         private void CreateWorktree()
         {
             //https://git-scm.com/docs/git-worktree
-            var arguments = "worktree add " + WorktreeDirectory;
+            var arguments = "worktree add \"" + WorktreeDirectory + "\"";
```

Comment:
I hope this won't become another [_"Illegal characters in path" exception in worktrees_ issue...](https://github.com/gitextensions/gitextensions/issues/3489)

Reply:
Not sure of what I can do to work around that...

Reply:
There is nothing to be done. The cited issue does not apply here.

Reply:
OK, that was my feeling but was not sure. Thanks a lot.